sdks/python/tox.ini

```diff
@@ -90,13 +91,14 @@ commands =
   python setup.py test
 passenv = TRAVIS*
 
-[testenv:lint]
```

Comment:
Why was the name changed?
Please revert or update https://beam.apache.org/contribute/contribution-guide/

Reply:
Sure, I'll ping you on the PR for that.